src/main/java/codeu/model/data/Action.java

```diff
@@ -0,0 +1,26 @@
+package codeu.model.data;
+
+/** This a class of different actions that can be displayed as an activity. */
+public enum Action {
+  JOIN("RegisteringUser", true),
```

Comment:
These enum values don't seem particularly clear.  I think you should just call them REGISTER_USER, SEND_MESSAGE, and CREATE_CONVERSATION (or CREATE_CONV since that's fairly unambiguous.)